docs/conf/eu/2017/sponsorship/prospectus.rst

```diff
@@ -189,6 +189,12 @@ Sponsorship of the official party includes:
 - Mention of company on schedule for the party, and an on-stage thanks
 - Your logo on a banner or roll up at the party venue (provided by sponsor)
 
+Child Care
+-----------
+
+Many of our attendees have children, sponsoring day care makes it  easier for
```

Comment:
True, but maybe we can add more details like "exact sponsorship amount TBD based on demand, but would probably be in the hundreds" or something more elegant like that ;)